src/Engine/Utils.java

```diff
@@ -357,9 +407,12 @@ private static void expandSearchNode(Army team, MoveType fff, GameMap map, Searc
    * Utility class used for pathfinding. Optionally holds a
    *   reference to a parent node for path reconstruction.
    */
-  private static class SearchNode
+  public static class SearchNode extends XYCoord
   {
-    public int x, y;
+    private static final long serialVersionUID = 2637721435469761667L;
+    // Both XYCoord's and these are final, so no desync is possible.
+    // However, TODO: delete one pair or the other
+    final public int x, y;
```

Comment:
TODO When?

Reply:
I'm thinking I want to get everything else merged, and then just do it in this PR.